release/blocker_tracker_script.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+
+JIRA_TOKEN=""
+SLACK_WEB_HOOK_URL=""
+
+function main() {
+    local blocker_issues_keys=($(curl --request GET --url 'https://liferay.atlassian.net/rest/api/3/search?jql=project%20%3D%20%22PUBLIC%20-%20Liferay%20Product%20Delivery%22%20and%20labels%20%3D%20release-blocker%20and%20status%20%21%3D%20Closed&fields=issuekey' --user "jira-cloud-enterprisereleasehu@liferay.com:<$JIRA_TOKEN>" --header 'Accept: application/json' | jq -r '.issues[].key'))
+
+    cd liferay-portal || (echo "Failed to change directory to liferay-portal"; exit)
+
+    local git_pull_response
+
+    git_pull_response=$(git pull origin master)
+
+    if [[ "${git_pull_response}" == *"Already up to date"* ]]
+    then
+        echo "The master branch hasn't been updated since the last run, so no changes in the blocks have occurred"
```

Comment:
@kiwm, please fix the spelling of the word `blockers` and add a `.` to the end of the last sentence. :wink: